packages/web/components/Player/Section/PlayerHero.tsx

```diff
@@ -95,90 +78,46 @@ export const PlayerHero: React.FC<Props> = ({
             isRound
             _active={{
               transform: 'scale(0.8)',
-              backgroundColor: 'transparent',
+              bg: 'transparent',
             }}
           />
         </Box>
       )}
-      <Box textAlign="center" mb={8} mt={2}>
+      <Box align="center" mb={8} mt={2}>
         <PlayerAvatar
-          w={{ base: 32, md: 56 }}
-          h={{ base: 32, md: 56 }}
+          w="min(var(--chakra-sizes-56), 100%)"
```

Comment:
This might be causing some issues as seen in the screenshot below. ideally we want it to be a circle ?
![Screenshot from 2022-02-28 16:22:59](https://user-images.githubusercontent.com/10012209/155971064-7fcbcce6-2057-4ff3-986f-005de07b6b06.png)

Reply:
Yeah, my fix for elements with default values wasn't quite right.

It's now keeping the default values (emoticons in this case) from loading.

I'm going to get back to it shortly…